rs/execution_environment/src/canister_manager/types.rs

```diff
@@ -646,6 +655,20 @@ impl AsErrorHelp for CanisterManagerError {
                         .to_string(),
                 doc_link: doc_ref("canister-snapshot-invalid-ownership"),
             },
+            CanisterManagerError::CanisterSnapshotNotController { .. } => {
+                ErrorHelp::UserError {
+                    suggestion: "Only a controller of the canister that the snapshot belongs to can load it."
```

Comment:
This is already part of the error message so it will be repeated, right?

Reply:
Good point. I'll rephrase to "make sure you're a controller of the canister that owns this snapshot".